src/main/java/io/vertx/core/http/impl/HttpClientImpl.java

```diff
@@ -55,7 +55,6 @@
   private final ConnectionManager connectionManager;
   private final Closeable closeHook;
   private final SSLHelper sslHelper;
-  final HttpClientMetrics metrics;
```

Comment:
Out of curiosity, was there a particular reason for moving the HttpClientMetrics field to the connection manager? Wondering why the move since the lifecycle was already managed here.